InWorldz/InWorldz.Data.Inventory.Cassandra/LegacyMysqlStorageImpl.cs

```diff
@@ -107,15 +107,15 @@ public InventoryFolderBase findUserTopLevelFolderFor(UUID owner, UUID folderID)
         /// <returns></returns>
         public List<InventoryItemBase> getItemsInFolders(IEnumerable<InventoryFolderBase> folders)
         {
-            string inList = "";
+            string inList = String.Empty;
 
             foreach (InventoryFolderBase folder in folders)
             {
-                if (inList != "") inList += ",";
+                if (inList != String.Empty) inList += ",";
                 inList += "'" + folder.ID.ToString() + "'";
             }
 
-            if (inList == "") return new List<InventoryItemBase>();
+            if (String.IsNullOrEmpty(inList)) return new List<InventoryItemBase>();
```

Comment:
No semantic change: `inList` cannot be null.  Might have been ok to do `inList.Length ==0` instead.